script.js

```diff
@@ -195,4 +195,43 @@ form.addEventListener('submit', (event) => {
   } else {
     form.submit();
   }
-});
\ No newline at end of file
+});
+
+// ######## Local storage ########
+
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return (
+      (e instanceof DOMException && e.code === 22)
+      || e.code === 1014 || e.name === 'QuotaExceededError'
+      || (e.name === 'NS_ERROR_DOM_QUOTA_REACHED' && storage && storage.length !== 0)
+    );
+  }
+}
+
+if (storageAvailable('localStorage')) {
+  const inputs = [form.name, form.email, form.message];
+  inputs.forEach((input) => {
+    input.addEventListener('input', () => {
+      const storagData = {
+        fullName: form.name.value,
+        email: form.email.value,
+        message: form.message.value,
+      };
+      localStorage.setItem('data', JSON.stringify(storagData));
+    });
+  });
+  const getData = JSON.parse(localStorage.getItem('data'));
```

Comment:
Nice 🔥 The form data for all inputs are stored in a single object, saved to a single local storage registry 💪😎